packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -685,38 +846,19 @@ describeIf(process.platform !== 'win32' && !isMacOrWindowsCI)('MongoDB', () => {
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
   })
 
-  test('re-introspection should error (not supported)', async () => {
+  test('re-introspection should error (not supported) (existing models)', async () => {
     ctx.fixture('schema-only-mongodb')
     const introspect = new DbPull()
-    await introspect.parse(['--schema=./prisma/no-model.prisma'])
-    // now re-introspection
-    const result = introspect.parse(['--schema=./prisma/no-model.prisma'])
+    const result = introspect.parse([])
     await expect(result).rejects.toThrowErrorMatchingInlineSnapshot(`
             Iterating on one schema using re-introspection with db pull is currently not supported with MongoDB provider (Preview).
             You can explicitely ignore and override your current local schema file with prisma db pull --force
             Some information will be lost (relations, comments, mapped fields, @ignore...), follow https://github.com/prisma/prisma/issues/9585 for more info.
```

Comment:
This is still logged though, right? Would it show up in `expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(...)`?